internal/validations/clusterlogforwarder/validate_clusterlogging_dependency.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func ValidateClusterLoggingDependency(clf loggingv1.ClusterLogForwarder, k8sClient client.Client, extras map[string]bool) (error, *loggingv1.ClusterLogForwarderStatus) {
 	if clf.Name == constants.SingletonName && clf.Namespace == constants.OpenshiftNS && !extras[constants.ClusterLoggingAvailable] {
-		return errors.NewValidationError("ClusterLogForwarder '%s/%s' is dependent on a ClusterLogging instance", constants.OpenshiftNS, constants.SingletonName), nil
+		return errors.NewValidationError("ClusterLogForwarder '%s/%s' is dependent on a ClusterLogging instance with a valid Collector configuration", constants.OpenshiftNS, constants.SingletonName), nil
```

Comment:
lets say... with a valid spec.collector

maybe makes it more obvious where exactly to fix your CL